libparsec/crates/client/tests/unit/workspace/file_operations_stateful.rs

```diff
@@ -160,7 +160,10 @@ impl StateMachineTest for FileOperationStateMachine {
         ref_state: &<Self::Reference as ReferenceStateMachine>::State,
     ) {
         // 1. Manifest integrity
-        state.manifest.assert_integrity();
+        state
+            .manifest
+            .check_content_integrity()
+            .expect("Manifest integrity");
```

Comment:
I'd stick with `unwrap()` here given we only use `.unwrap()` in the tests so it is obvious this is a test assert  (while using `.expect()` can lead to thing the test itself is broken)